File: Minimum Deletions to make string balanced.py
class Solution(object):
    def minimumDeletions(self, s):
        """
        :type s: str
        :rtype: int
        """
        b = 0
        ans = 0
        for i in s:
            if i == "b":
                b = b + 1
            else:
                ans = ans + 1
                ans = min(ans ,b)

        return ans 


        

File: Minimum Deletions to make String Balanced.java
class Solution {
    public int minimumDeletions(String s) {
        int b = 0;
        int ans = 0;
        for(int i = 0; i< s.length();i++){
            char ch = s.charAt(i);
            if (ch == 'b'){
                b++;
            } else {
                ans++;
                ans = Math.min(ans,b);
            }
        }
        return ans;
    }
}
